rpcclient/wallet.go

```diff
@@ -803,9 +803,14 @@ func (c *Client) PurchaseTicketAsync(fromAccount string,
 		ticketFeeFloat = ticketFee.ToCoin()
 	}
 
+	splitTxVal := uint32(1)
+	if splitTx != nil {
+		splitTxVal = *splitTx
+	}
+
 	cmd := dcrjson.NewPurchaseTicketCmd(fromAccount, spendLimit.ToCoin(),
 		&minConfVal, &ticketAddrStr, &numTicketsVal, &poolAddrStr,
-		&poolFeesFloat, &expiryVal, nil, noSplitTransaction, &ticketFeeFloat)
+		&poolFeesFloat, &expiryVal, dcrjson.String(""), &splitTxVal, &ticketFeeFloat)
```

Comment:
No need to allocate:
```Go
dcrjson.String("") -> nil
```

Reply:
This was actually in necessary change since the arguments that follow a `nil` pointer will not be processed.  The workaround is to have a non-nil pointer to an empty string.